arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-two</team>
+        </all>
+    </deny>
+    <all id="red-team-win">
+        <variable var="red_team_win">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win">
+        <variable var="blue_team_win">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-win-final">
+        <variable var="red_team_win">2</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win-final">
+        <variable var="blue_team_win">2</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-lose">
+        <variable var="red_team_lose">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-lose">
+        <variable var="blue_team_lose">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-final">
+        <variable var="red_team_win">5</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-final">
+        <variable var="blue_team_win">5</variable>
+        <team>team-two</team>
+    </all>
+    <any id="remove-entities">
+        <entity>dropped item</entity>
+        <entity>boat</entity>
+    </any>
+</filters>
+<variables>
+    <variable id="round_number" scope="match" default="1"/>
+    <variable id="round_state" scope="match" default="9"/> <!-- running state, skip cages first round -->
+    <variable id="scoring_team" scope="match" default="0"/>
+    <score id="team_score"/>
+    <with-team id="red_team_score" var="team_score" team="team-one"/>
+    <with-team id="blue_team_score" var="team_score" team="team-two"/>
+    <variable id="player_score" scope="player" default="0"/>
+    <variable id="red_team_win" scope="match" default="0"/>
+    <variable id="red_team_lose" scope="match" default="0"/>
+    <variable id="blue_team_win" scope="match" default="0"/>
+    <variable id="blue_team_lose" scope="match" default="0"/>
+    <variable id="gameover" scope="match" default="0"/>
+</variables>
+<actions>
+    <!-- show to Red team on a round Win -->
+    <trigger filter="red-team-win" action="red-team-win-action" scope="player"/>
+    <action id="red-team-win-action" scope="player">
+        <switch-scope outer="player" inner="match">
+            <switch-scope outer="match" inner="team" filter="only-team-one">
+                <set var="red_team_win" value="0"/>
+                <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
+                <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
+                <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>
+
+                <sound key="fireworks.blast_far" volume="1" pitch="1"/>
+                <sound key="fireworks.twinkle" volume="1" pitch="1"/>
+                <sound key="fireworks.twinkle" volume="1" pitch=".8"/>
+                <sound key="objective_good" volume="1" pitch="1"/>
```

Comment:
you've defined this list of sounds like 10 times in this xml, consider extracting it to an action, eg:
```xml
    <action id="good-sounds" scope="team">
        <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
        <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
        <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>

        <sound key="fireworks.blast_far" volume="1" pitch="1"/>
        <sound key="fireworks.twinkle" volume="1" pitch="1"/>
        <sound key="fireworks.twinkle" volume="1" pitch=".8"/>
        <sound key="objective_good" volume="1" pitch="1"/>
    </action>
```

then wherever you want to play those, you can just reference it, eg:

```xml
             ...
            <switch-scope outer="match" inner="team" filter="only-team-one">
                <set var="red_team_win" value="0"/>
                <action id="good-sounds"/>
                <message fade-in="0.5s" stay="3s" fade-out="0.5s">
                ...

```

Reply:
Wow, I don't think I realized this was possible.  So much better, thanks.